lib/models/device.coffee

```diff
@@ -541,6 +659,21 @@ exports.restart = (uuid, callback) ->
 	.nodeify(callback)
 
 ###*
+# @summary Restart application on device. *Deprecated*: Use
```

Comment:
Can you remove the deprecation notice and link from the summary? JSDoc is already marking it as deprecate because of the `@deprecated` tag, and instead of linking to the other function directly, you can make use of `@see`.